pilot/util/harvester.py

```diff
@@ -46,6 +51,7 @@ def get_job_request_file_name():
     :return: job request file name.
     """
 
+    #logger.debug('config.Harvester.__dict__ : {0}'.format(config.Harvester.__dict__))
     return join(environ['PILOT_HOME'], config.Harvester.job_request_file)
```

Comment:
did you mean os.path.join?

Reply:
yes change to os.path.join for readability